src/olympia/accounts/tests/test_serializers.py

```diff
@@ -250,6 +251,21 @@ def test_validate_homepage(self):
             # It should not raise when value is allowed.
             assert serializer.validate_homepage(allowed_url) == allowed_url
 
+    def test_site_status(self):
+        data = super(TestUserProfileSerializer, self).test_basic()
+        assert data['site_status'] == {
+            'read_only': False,
+            'notice': None,
+        }
+
+        set_config('site_notice', 'THIS is NOT Á TEST!')
```

Comment:
We could maybe have one `assert` here to cover the case where the site is not in read only mode but there is a message to display to the users.

Reply:
https://github.com/mozilla/addons-server/pull/12356/commits/4acd93a499ade786efc4967b45807f2550d4a248